libbeat/processors/condition.go

```diff
@@ -399,6 +405,17 @@ func (c *Condition) checkRange(event ValuesMap) bool {
 	return true
 }
 
+func (c *Condition) checkHasFields(event ValuesMap) bool {
+	for _, field := range c.hasfields.fields {
+		_, err := event.GetValue(field)
+		if err != nil {
+			logp.Info("Field %v not present in the event", field)
```

Comment:
I think it must be close to exists code, like in checkEquals, in case if there is no value, just return false.
There is no info logging at all, is it make sense to introduce it only to this case?

Reply:
Thanks for the comments @ewgRa . I have made the changes.